File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/ConvertMyEnumToString.cs
/// <filename>
///     ConvertMyEnumToString.cs
/// </filename>
using dependencies;

namespace dependencies
{
    public class ConvertMyEnumToString
    {
        public static string Execute(dependencies.MyEnum symbol)
        {
            if (symbol == dependencies.MyEnum.A)
            {
                return "A";
            }

            if (symbol == dependencies.MyEnum.B)
            {
                return "B";
            }

            return "?";
        }
    }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/convertMyEnumToString.cpp
#include "convertMyEnumToString.h"
#include <string>
#include "MyEnum.h"

namespace dependencies
{
  std::string convertMyEnumToString(MyEnum symbol)
  {
    if (symbol == dependencies::MyEnum::A)
    {
      return std::string("A");
    }

    if (symbol == dependencies::MyEnum::B)
    {
      return std::string("B");
    }

    return std::string("?");
  }
}
